docker_compose_swarm_mode.py

```diff
@@ -214,9 +214,23 @@ def start(self, services=None):
 
 
 def main():
+    for k, v in {
+        'COMPOSE_FILE': 'docker-compose.yml',
+        'COMPOSE_HTTP_TIMEOUT': '60',
+        'COMPOSE_TLS_VERSION': 'TLSv1'
+    }.iteritems():
+        if not k in os.environ:
+            os.environ[k] = v    
+
+    env_path = os.path.join(os.getcwd(), '.env')
+
+    if os.path.isfile(env_path):
+        with open(env_path) as env_file:
+            map(lambda e: os.environ.update({e[0]:e[1]}), (line.strip().split('=', 1) for line in env_file if not line.startswith('#') and line.strip()))
```

Comment:
According to the [docs](https://docs.docker.com/compose/env-file/):
> Values present in the environment at runtime will always override those defined inside the .env file

So we should only call `update` if there was no such a key.

I guess your idea was to make `.env` override the initial `COMPOSE_FILE`, ... map.
Then we should first merge that map with the content of the `.env` file and after that update the environment.